src/interactivity/onecore/precomp.h

```diff
@@ -9,7 +9,7 @@
 #include <ntrtl.h>
 #include <nturtl.h>
 #include <windows.h>
-#include "wchar.h"
+#include "cwchar"

```

Comment:
These are also code from outside the repo, I suppose.

Reply:
Is that so?